haskell/nettest/Nettest.hs

```diff
@@ -67,11 +67,14 @@ scNettestScenario originateTransferlist transferlistType = uncapsNettest $ do
   comment "Originating metadata registry contract"
   mrAddress <- nettestOriginateMetadataRegistry
 
+  comment "Originating contract metadata contract"
```

Comment:
`contract-metadata` was renamed to `tzip16-metadata`, but in code we still say "contract metadata" in many places. Let's not change it for now, but when we do #131 and remove the legacy interface, we will have only one metadata contract and will call it just "metadata" without "contract" or "tzip-16" prefix.